circle.yml

```diff
@@ -12,3 +12,4 @@ dependencies:
 test:
   override:
     - py.test -r w tests --junitxml=$CIRCLE_TEST_REPORTS/temporal-sqlalchemy.xml
+    - flake8
```

Comment:
flake8 defaults to <80, which is to say max 79.

This conforms with PEP8.

The general rule of thumb I tend to hear is: open source should stick to 80 characters but private projects might want to raise the limit to 100 or 120, depending on team preferences.

Personally, I am inclined to advocate for strict PEP8 adherence for open source projects.